device/src/bin/v2.rs

```diff
@@ -650,17 +639,23 @@ fn panic(info: &core::panic::PanicInfo) -> ! {
         None => write!(&mut panic_buf, "{}", info),
     };
 
-    let spi = Spi::new(peripherals.SPI2, 80u32.MHz(), SpiMode::Mode2, &clocks)
-        .with_sck(io.pins.gpio8)
-        .with_mosi(io.pins.gpio7);
+    let spi = Spi::new_with_config(
+        peripherals.SPI2,
+        spiConfig {
+            frequency: 40.MHz(),
```

Comment:
Why did this change from 80 to 40. Did the semantics of the arguments change or are you changing the frequency for some reason?

Reply:
This should be 80 my bad